tensorflow_datasets/scripts/templates/dataset.mako.md

```diff
@@ -165,6 +165,31 @@ ${builder.info.citation}
 % endif
 </%def>
 
+<%def name="display_figure(builder)">\
+<%
+  import requests
+  import os
+
+  config_name = os.path.split(builder.info.full_name)[-2]
+  ds_name_config = config_name.replace("\\", "/").replace("/", "_")
```

Comment:
Replace `.replace("/", "-")` to avoid potential colisions `my_dataset_xyz` `my_dataset/xyz`

Reply:
okay